### PR TITLE
corrected USNC acronym

### LINKED ISSUE #3: Review of Poster
I have updated the poster to the 3 slide format requested by the conference (https://www.ans.org/meetings/student2021/newresources/). 

A pdf of the poster is here: 
[poster.pdf](https://github.com/arfc/2021-bachmann-ans-student/files/6191450/poster.pdf). The figures probably need some editing to allow them to be larger, but I'm not sure how to do that. I am assuming the slides do not need to stand on their own, so much of the content will be explained by me when I record the video.

### PR DESCRIPTION
**Pull request by abachma2**
This PR is from a new fork of the repo, since my previous fork broke when I made this repo public. This PR will include the changes made for PR #5 (closed because I deleted that repo) and additional change made while recording the presentation. 

An updated version of the poster is: 
[poster.pdf](https://github.com/arfc/2021-bachmann-ans-student/files/6359442/poster.pdf)

### CODE REVIEW

File: poster/slides.tex [hunk 1]
```diff
@@ -86,12 +86,13 @@
         \item Neither of the transition scenarios (scenarios 2 and 3) meet the desired power level.
         \item Scenario 2 requires a higher mass of \gls{HALEU} than scenario 3.
```
Comment (npanczyk) [inline poster/slides.tex:87]:
```suggestion
        \item Scenario 2 requires a higher mass of \gls{HALEU} than Scenario 3.
```

File: poster/slides.tex [hunk 2]
```diff
@@ -86,12 +86,13 @@
         \item Neither of the transition scenarios (scenarios 2 and 3) meet the desired power level.
```
Comment (npanczyk) [inline poster/slides.tex:86]:
```suggestion
        \item Neither of the transition scenarios (Scenarios 2 and 3) meet the desired power level.
```

**Review by npanczyk — APPROVED** [review]
Changes look good to me! Couple notes on the pdf:

- [x] "Scenario" should be capitalized in Fig. 2 whenever you refer to a specific scenario
- [x] "Year" should also be capitalized in Fig. 2
- [x] spacing on the table is a little awkward (it seems like there are extremely long gaps that wrap over to the next line), totally optional to change though, okay as is

Let me know if there's anything else I should look at!

**Comment by abachma2** [comment]
I can make those changes, but what I have in this PR is what was submitted for the conference. What are your thoughts @katyhuff ?

**Comment by abachma2** [comment]
Made the changes.

**Review by munkm — APPROVED** [review]
I think this looks great and am happy with it as-is. However, if you do want to do a few minor adjustments I noticed the following:
- The caption of Figure 4 is going off the page slightly
- You use Xe-100^{TM} in Table 1 but there is no TM used later (e.g. in your description for Scenario 3, in the legend of Figure 2)

**Review by munkm — APPROVED** [review]

**Pull request merged**